src/main/java/org/example/controllers/ApplicationController.java

```diff
@@ -25,7 +25,7 @@ public ApplicationController(final ApplicationService applicationService) {
     }
 
     @POST
-    @Path("/apply-for-role")
+    @Path("/applyForJobRole")
```

Comment:
any reason for changing this? could it break any automated tests?

Reply:
was keeping naming style consistent. automated tests have been updated to pass with this.